### PR TITLE
Storybook Theme: Sage-ify Storybook

### PR DESCRIPTION
**Pull request by monicawheeler**
## Description

Updates for the favicon and logo to remove the storybook branding.

### Changes
* Add favicon
* Add Sage logo

## Screenshots

|  Before  |  After  |
|--------|--------|
| before | after |

## Testing in `sage-lib`
1. Navigate to http://localhost:4100/?path=/docs/introduction--page
2. View the logo change and favicon update

## Testing in `kajabi-products` (KP)
1. (**LOW**) - No KP testing necessary

### CODE REVIEW

**Review by ju-Skinner — CHANGES_REQUESTED** [review]
Per our discussion via Huddle. We do want some element of surprise when releasing Pine. So we will remove the themeing of Sage's storybook, but keep the Favicon and Logo.

File: packages/sage-react/.storybook/SageTheme.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  colorSecondary: '#040506',
+
+  // UI
+  appBg: '#f9fafa',
+  appContentBg: 'white',
+  appBorderColor: '#eceeef',
+  appBorderRadius: 6,
+
+  // Text colors
+  textColor: 'black',
```
Comment (pixelflips) [inline packages/sage-react/.storybook/SageTheme.js:16]: Not a blocker whatsoever, but any thoughts on a slightly lighter shade? https://uxmovement.com/content/why-you-should-never-use-pure-black-for-text-or-backgrounds/

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼 - Nice work!

**Review by ju-Skinner — APPROVED** [review]
Great job. :shipit:

**Review by anechol — APPROVED** [review]

**Pull request merged**